ansible/roles/nomad/tasks/main.yml

```diff
@@ -116,11 +118,22 @@
   loop:
     - nomad
     - client
+    - client_vault
   notify: nomad_restart
   tags:
     - nomad
   when: "'client' in inventory_hostname"
 
+- name: Insert Vault token into nomad vault integration agent config.
+  replace:
+    path: /etc/nomad.d/server_vault.hcl #"{{ item.path }}"
+    regexp: "REPLACETOKEN"
```

Comment:
I don't really know Ansible, but I would think we would no longer need this given the plan to use environment variable VAULT_TOKEN